src/units/unit.hpp

```diff
@@ -1054,19 +1054,21 @@ class unit : public std::enable_shared_from_this<unit>
 	 * @param loc The unit's location (to resolve [resistance] abilities)
 	 * @param weapon The weapon to check for any abilities or weapon specials
 	 * @param opp_weapon The opponent's weapon to check for any abilities or weapon specials
+	 * @param for_tooltip if true, the function effective_damage_type shouldn't be called same if opp_weapon exist.
```

Comment:
```suggestion
	 * @param for_tooltip if true, the function effective_damage_type shouldn't be called if opp_weapon exist.
```
IIUC, the condition checks for the existence of `opp_weapon` only, so `same` is not needed.

Reply:
fixed